rpc/rpc.h

```diff
@@ -234,7 +234,8 @@ namespace rpc
     };
 
```

Comment:
RPC server can specify IPv6 by using a IPv6 socket, but client can't. So add this flag.